frontend/app/src/components/settings/api-keys/ExchangeKeysForm.vue

```diff
@@ -281,6 +267,59 @@ onMounted(() => {
 
 const krakenAccountTypes = KrakenAccountType.options;
 const exchanges = SUPPORTED_EXCHANGES;
+
+const sensitiveFieldEditable = computed(() => !get(edit) || get(editKeys));
+
+const rules = {
+  name: {
+    required: helpers.withMessage(
+      tc('exchange_keys_form.name.non_empty'),
+      requiredIf(computed(() => !get(edit)))
```

Comment:
This seems to be a prime use case for `requiredUnless` => `requiredUnless(edit)`
https://vuelidate-next.netlify.app/validators.html#requiredunless